includes/connect/class-wc-stripe-connect.php

```diff
@@ -265,5 +302,80 @@ private function record_account_connect_track_event( bool $had_error ) {
 			// a queue wouldn't be processed due to the redirect that comes after.
 			WC_Tracks::record_event( $event_name, [ 'is_test_mode' => $is_test ] );
 		}
+
+		/**
+		 * Schedules the App OAuth connection refresh.
+		 *
+		 * @since 8.6.0
+		 */
+		private function schedule_connection_refresh() {
+			if ( ! $this->is_connected_via_app_oauth() ) {
+				return;
+			}
+
+			/**
+			 * Filters the frequency with which the App OAuth connection should be refreshed.
+			 * Access tokens expire in 1 hour, and there seem to be no way to customize that from the Stripe Dashboard:
+			 * https://docs.stripe.com/stripe-apps/api-authentication/oauth#refresh-access-token
+			 * We schedule the connection refresh every 50 minutues.
```

Comment:
This says 50 minutes. But the scheduled event is 1 hour - 5 minutes so 55 minutes.

Reply:
Nice catch 🤦🏼 
```suggestion
			 * We schedule the connection refresh every 55 minutues.
```